app/src/main.ts

```diff
@@ -190,22 +190,25 @@ if (shouldQuit) {
     }
   });
 
-  if (appArgs.widevine !== undefined) {
```

Comment:
This condition was excessive in a normal build. In a normal build, appArgs.widevine is `false`. So, this branch was taken even out of a non-widevine build, and since `widevine-ready` isn't called here, `onReady` was never called.

Reply:
@TheCleric asking for a PR to be sure this is the fix you'd expect, or if `appArgs.widevind` shouldn't be `false` at this point.

Reply:
Nope looks good to me. Sorry for the bug!